### ISSUE TITLE
Separate apps for image capturing and networking

### ISSUE THREAD

**Issue by polarbearing** [mapillary/mapillary_issues#1416]
Please separate apps for image capturing and networking. Reasons:

- app apparently consumes too many resources which might be a reason for the crashes such as #724. Splitting the two main functions will make the app leaner, and hopefully more stable.
- when used for capturing in a vehicle, the network functions are unnecessary and distracting. E.g. when the app starts, it tries for Feed information and does not get any. A small button has to be pressed to reach the camera, when the camera button is missed because the vehicle is in motion, it tries to load a browser, again pointless since there is no connection.

**Comment by jesolem** [comment]
Is there any indication that too many resources crashes the app? Do you have logs @polarbearing ?

**Comment by polarbearing** [comment]
I regularly send logs @peterneubauer along with issue comments.

**Comment by nicolas17** [comment]
Is this specific to the Android app or does it apply to iOS too? (There is no github issue tag specifying it)
I'm on iOS and don't have this kind of problem. For example, the app starts in the Feed tab but shows cached information from the last time I opened it while connected.

**Comment by jesolem** [comment]
Is this still an issue we should track? We are looking at making a "light" version of Android for old devices.

**Comment by polarbearing** [comment]
Yes please continue with the 'capture only' app, which would be very helpful for getting a quick start while driving. It should also close completely when the app is closed and not squat in the background, still consuming memory.

**Comment by polarbearing** [comment]
The memory footprint of the android app is 10 times compared to the early apps, and has again increased 50% from major release 2 to 3. 

As the web uploader accepts pictures from the app (which in the past was not the case), my workflow is now to transfer the pictures to the laptop for large-screen review, and upload from there. Thus I don't need any feed or upload information on the mobile device, but faster handling/starting there.

**Comment by ovivoda** [comment]
_mapillary_issues is no longer an active repository._

_Let’s continue discussing features in our forum! We will import this issue as topic to the <a href="http://forum.mapillary.com ">Mapillary Forum</a> in the coming weeks. We’ll notify here once it’s been done. See you there!_
